### PR TITLE
Add Boostcamp and Shazam as Easy

### PR DESCRIPTION
**Pull request by Szythers**
Added sections for Boostcamp and Shazam, hopefully I did it right this is my first time using Github and I just copy and pasted other easy entries and replaced the data.

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -2642,6 +2642,15 @@
         ]
     },
 
+    {
+        "name": "Boostcamp",
+        "url": "https://www.boostcamp.app/data-deletion-request",
+        "difficulty": "easy",
```
Comment (Kefaku) [inline _data/sites.json:2648]:
```suggestion
        "difficulty": "easy",
        "notes": "Visit the linked page and confirm the deletion.",
```

File: _data/sites.json [hunk 2]
```diff
@@ -16204,6 +16213,15 @@
         ]
     },
 
+    {
+        "name": "Shazam",
+        "url": "https://www.shazam.com/privacy/login/delete",
+        "difficulty": "easy",
```
Comment (Kefaku) [inline _data/sites.json:16219]:
```suggestion
        "difficulty": "easy",
        "notes": "Visit the linked page and sign in to your account. Your account will be deleted within 30 days. According to their privacy page \"Shazam may retain data that has been deidentified and is no longer associated with you.\"",
```

**Review by Kefaku — COMMENTED** [review]
I always prefer having comments to guide the user, even if there isn't much additional information like for Boostcamp.
At least Shazam should have notes giving additional information like accounts beeing deleted within 30 days and Shazam possibly retaining some data.

I therefore added some notes as suggested changes.

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contributions!

**Pull request merged**